### PR TITLE
[utility] Make content in hanging paragraph non-normative.

### PR DESCRIPTION
**Pull request by jensmaurer**
Partially addresses #1771.

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -35,13 +35,20 @@
...
+\end{note}
 
 \indexhdr{utility}%
 \indexlibrary{\idxcode{rel_ops}}%
 \rSec2[utility.syn]{Header \tcode{<utility>} synopsis}
 
+\pnum
+The header \tcode{<utility>} defines several types and function templates
+that are described in this Clause. It also defines the template \tcode{pair}
+and various function templates that operate on \tcode{pair} objects.
```
Comment (tkoeppe) [inline source/utilities.tex]: What on earth is the special call-out to `pair` achieving here? The header also defines `in_place_index` and `index_sequence_for`. Maybe this sentence came from an older time when `pair` really was the only class template here?

Comment (jensmaurer) [reply source/utilities.tex]: This is pre-existing text that just got moved (from below the synopsis). There are similar introductory notes for most of the header synopses. If you feel this particular intro text warrants improvement, feel free to suggest something in a separate pull request.

Comment (tkoeppe) [reply source/utilities.tex]: OK, let me just keep this open so as to not forget.

@zygoloid, @jwakely: do you think we should list all the other class templates explicitly in this strange little sentence, or do something more radical (such as removing it altogether)?

**Comment by jensmaurer** [comment]
Rebased.

File: source/utilities.tex (hunk 1 above)
Comment (zygoloid) [reply source/utilities.tex]: I'd prefer we just drop this entirely. If someone wants to know what's in <utility>, that's what the synopsis is for. I don't see any point in including a paragraph that says "The header <utility> includes the stuff we say it includes in the synopsis."

Comment (zygoloid) [reply source/utilities.tex]: Maybe we could ditch this and move the hanging paragraph into this section, replacing "This subclause" with "The header \<utility>".

**Comment by zygoloid** [comment]
Merged with @tkoeppe's suggestion and committed as 63dd5c67.

**Pull request closed without merging**